my-project/public/js/ProductCategoryScripts/Shared.js

```diff
@@ -24,11 +30,49 @@ function RemoveImageFromCategory(path, img)
         })
         .then((response)=>{
             img.remove();
-            alert(response.message);
+            BootstrapAlertSuccess(response.message);
```

Comment:
@mSobonVirtua we can make one method out of those 2, with second param containing type of message